src/System.ServiceModel.Security/ref/System.ServiceModel.Security.cs

```diff
@@ -234,10 +234,19 @@ public partial class GenericXmlSecurityToken : System.IdentityModel.Tokens.Secur
             SecurityKeyIdentifierClause internalTokenReference,
             SecurityKeyIdentifierClause externalTokenReference,
             System.Collections.ObjectModel.ReadOnlyCollection<System.IdentityModel.Policy.IAuthorizationPolicy> authorizationPolicies) {}
-        public override string Id { get; }
-        public override DateTime ValidFrom { get; }
-        public override DateTime ValidTo { get; }
-        public override System.Collections.ObjectModel.ReadOnlyCollection<System.IdentityModel.Tokens.SecurityKey> SecurityKeys { get; }
+        public override string Id => default;
+        public override DateTime ValidFrom => default;
+        public override DateTime ValidTo => default;
+        public SecurityKeyIdentifierClause InternalTokenReference => default;
+        public SecurityKeyIdentifierClause ExternalTokenReference => default;
+        public System.Xml.XmlElement TokenXml => default;
+        public SecurityToken ProofToken => default;
+        public System.Collections.ObjectModel.ReadOnlyCollection<System.IdentityModel.Policy.IAuthorizationPolicy> AuthorizationPolicies => default;
+        public override System.Collections.ObjectModel.ReadOnlyCollection<System.IdentityModel.Tokens.SecurityKey> SecurityKeys => default;
+        public override bool CanCreateKeyIdentifierClause<T>() => default;
+        public override T CreateKeyIdentifierClause<T>() => default;
+        public override bool MatchesKeyIdentifierClause(SecurityKeyIdentifierClause keyIdentifierClause) => default;
```

Comment:
Aren't these overrides just re-defining the same results inherited from SecurityToken?

Reply:
Yes, but I think it might do something with regards to what happens when a derived class calls base.MatchesKeyIdentifierClause. When calling a base method, it can't just look up in the vtable for the abstract type and find the method, it's got to look up the method in the actual base class implementation. At least semantically there's a difference. If you didn't know this class also overrode it, you might expect it to call the implementation in the parent class of this. I suspect it would resolve correctly at runtime regardless, but semantically this is more correct.